src/middleware/router.js

```diff
@@ -657,7 +667,8 @@ function sendKafkaRequest(ctx, route) {
           method: ctx.request.method,
           path: ctx.request.url,
           headers: ctx.request.headers,
-          body: ctx.body && ctx.body.toString()
+          requestBody: ctx.reqBody ?? '',
+          responseBody: ctx.resBody ?? ''
```

Comment:
I'm wondering what else we might need in the future from the response. Perhaps we need to send a structure that similar to what our orchestration's capture (https://github.com/jembi/openhim-core-js/pull/1197/files#diff-65ff1233d04bd7dce50449a41b15b0e5f5dc5226fc06c08dd9fbde9305961c02R482)

This would also introduce a breaking change by getting rid of the `body` property. Perhaps we should keep that to keep backward compatibility.